third_party/tizen/tizen_dev_certificate.py

```diff
@@ -28,20 +28,33 @@
 logging.basicConfig(level=logging.DEBUG)
 
 
+def run(cmd):
+    logging.debug("Run: %s", " ".join(cmd))
```

Comment:
consider shelex escaping instead of just space-join, so we guarantee copy & paste of command that is run is always ok.

Reply:
Thanks for the tip!